src/settings/LoanPolicy/components/EditSections/RenewalsSection/RenewalsSection.js

```diff
@@ -125,7 +146,10 @@ class RenewalsSection extends React.Component {
             </div>
           </>}
         { policy.isRenewable() && policy.isDifferentPeriod() &&
-          <div data-test-renewals-alternate-fixed-due-date-schedule>
+          <div
+            data-test-renewals-alternate-fixed-due-date-schedule
+            data-testid="altFDDSTestId"
```

Comment:
maybe let us not add `TestId` postfix. we already have `data-testid` which says that it is testid. let us discuss.

Reply:
yeah, good idea. i'll do

Reply:
done